spec/factories/collections.rb

```diff
@@ -13,7 +13,6 @@
     required_license { 'MyString' }
     default_license { 'MyString' }
     email_when_participants_changed { false }
-    state { 'deposited' }
```

Comment:
Let it default to its default value, and use the object states traits in tests to vary the behavior, the same way we do with works.